README.md

```diff
@@ -60,9 +60,22 @@ The solution also includes a Continuous Integration/Continuous Delivery feature:
 
 # Installing Service Workbench
 
-## Accessing documentation using Docusaurus
+## Accessing Service Workbench Documentation
 
-Read the Service Workbench Deployment Guide provided in the installation. You can now also launch the Service Workbench docusaurus website. Please follow the steps in docs/README.md
+Service Workbench documentation can be accessed in the PDF format or by using the Docusaurus website. 
+
+### Documentation PDFs
+
+You can download the PDF versions if you do not have Service Workbench locally installed on your machine. Click the following links to download:
+
++ [Service Workbench Installation Guide](/docs/Service_Workbench_Installation_Guide.pdf)
++ [Service Workbench Configuration Guide](/docs/Service_Workbench_Configuration_Guide.pdf)
+
+**Note**: *Service Workbench Post Deployment Guide is currently accessible through Docusaurus and its PDF version will be available soon!*
+
+### Accessing Documentation using Docusaurus
+
+You first need to install Service Workbench on your local machine before accessing the documentation using Docusaurus. After installing Service Workbench, you need to install Docusaurus as well so that you can launch the help site. For steps on how to install Docusaurus using local installation, read the [Documentation Readme](/docs/README.md).
```

Comment:
Users don't have to install SWB to use docusaurus. We can just have this as the text for this section.
```
To view the Documentation in `/docs` correctly, please install `Docusaurus`. For steps on how to install Docusaurus locally please follow [the instructions here](/docs/README.md).
```

Reply:
Hi Tim, can we specify the other way to launch docusaurus without launching SWB. The reason why I have written this is because I don't have clarity on how to launch Docusaurus without installing SWB. I just got link from Stephen earlier which he created locally using his Isengard account. If there is any other way, I will appreciate your help in documenting that.